Library/Formula/libssh.rb

```diff
@@ -14,6 +14,7 @@ class Libssh < Formula
 
   depends_on "cmake" => :build
   depends_on "openssl"
+  depends_on "krb5" => :build unless OS.mac?
```

Comment:
Please change this to `homebrew/dupes/krb5`. Is it only a build time dependency? The libraries aren't needed at run time?